cli/module_loader.rs

```diff
@@ -63,6 +63,13 @@ impl CliModuleLoader {
     &self,
     specifier: &ModuleSpecifier,
   ) -> Result<ModuleCodeSource, AnyError> {
+    if specifier.as_str() == "node:module" {
```

Comment:
That would require to change "compat mode" logic to ignore this specifier, I think we should hold off on that until compat is removed. Since it's a built-in module I believe it should be excluded from an import map look up.